mobile_verifier/tests/rewarder_mappers.rs

```diff
@@ -0,0 +1,151 @@
+mod common;
+use crate::common::MockFileSinkReceiver;
+use chrono::{DateTime, Duration as ChronoDuration, Utc};
+use file_store::mobile_subscriber::{SubscriberLocationIngestReport, SubscriberLocationReq};
+use helium_crypto::PublicKeyBinary;
+use helium_proto::{
+    services::poc_mobile::{SubscriberReward, UnallocatedReward, UnallocatedRewardType},
+    Message,
+};
+use mobile_verifier::{reward_shares, rewarder, subscriber_location};
+use rust_decimal::prelude::*;
+use rust_decimal_macros::dec;
+use sqlx::{PgPool, Postgres, Transaction};
+use std::str::FromStr;
+use std::string::ToString;
+
+const SUBSCRIBER_1: &str = "subscriber1";
+const SUBSCRIBER_2: &str = "subscriber2";
+const SUBSCRIBER_3: &str = "subscriber3";
+const HOTSPOT_1: &str = "112NqN2WWMwtK29PMzRby62fDydBJfsCLkCAf392stdok48ovNT6";
+
+#[sqlx::test]
+async fn test_mapper_rewards(pool: PgPool) -> anyhow::Result<()> {
+    let (mobile_rewards_client, mut mobile_rewards) = common::create_file_sink();
+    let now = Utc::now();
+    let epoch = (now - ChronoDuration::hours(24))..now;
+
+    // seed db
+    let mut txn = pool.clone().begin().await?;
+    seed_mapping_data(epoch.end, &mut txn).await?;
+    txn.commit().await.expect("db txn failed");
+
+    tokio::select!(
```

Comment:
should this be a `tokio::join/try_join` instead? if it's a `select!` wouldn't it just exit when the first task completes and isn't rewarding the mappers necessarily the first step in order to trigger `receive_expected_rewards` from the mock sink client?

Reply:
tests are updated to use tokio join